src/install/local/manual-install.md

```diff
@@ -0,0 +1 @@
+# Automated Install
```

Comment:
Missing content or old artifact? Maybe autogenerated accidentally when updating SUMMARY.md

Reply:
Artifact of an orphaned link in Summary.md. This has been removed in my last commit :)